yt/go/yt/internal/params.go

```diff
@@ -2027,13 +2027,13 @@ func (p *ReadTableParams) AccessTrackingOptions() **yt.AccessTrackingOptions {
 type StartOperationParams struct {
 	verb    Verb
 	opType  yt.OperationType
-	spec    interface{}
```

Comment:
This is a generated code and the generator should be fixed as well.

One should not modify this file manually and always use generator.

Reply:
Remove changes from this file

Reply:
Why not fix generator?)

Reply:
We'd prefer to have all client code converted at once.